NewForce-at-Mountwest/bangazon-api-radvioli
# Payment Type Controller & Test

# Description

Created payment type controller which allows the user to create, edit, and delete payment types. I also created a test to make sure the controller is allowing the user to get a single payment type, get all payment types, create a payment type, edit a payment type, and delete a payment type.


## Related Ticket(s)
Ticket 3
Allow developers to access payment type resource



## Steps to Test
git add and git commit what you're working on on your branch
git checkout master to switch to your local master branch
git fetch --all will give you access to your teammate's branches (if they've pushed them up)
git checkout na-paymenttype to access my branch.
If you don't see the most recent version of their files, try git pull origin na-paymenttype
Run your server on their branch and make sure everything looks okay
Once server is running, use your browser to navigate to api/products to ensure that correct information is displayed.

Afterwards, in Visual Studio, go to the TestProduct.cs file under TestBangazonAPI and right-click on it and click on Run Tests. Ensure that all tests come back green.

Check verbiage and comments to make sure they are legible and understandable.


# Checklist:
